README.rst

```diff
@@ -29,38 +48,25 @@ Installation
         ftw.linkchecker
 
 
-One needs to add a config file (e.g. linkchecker_config.json) holding:
+Configuration
+=============
+
+A config file is required (there is an example in ``config.example``).
+In the config file following things can be configured:
 
 - portal path (unique identifier of the platform)
```

Comment:
It's not clear 
a) that portal path should be the key for a configuration dictionary
b) what the correct names of the other dictionary keys are